CMakeLists.txt

```diff
@@ -29,8 +29,11 @@ SET(CMAKE_LIBRARY_OUTPUT_DIRECTORY "${PROJECT_BINARY_DIR}/lib")
 SET(CMAKE_ARCHIVE_OUTPUT_DIRECTORY "${PROJECT_BINARY_DIR}/lib")
 
 IF(MSVC)
-  ADD_COMPILE_FLAG("/arch:sse2")
-  ADD_COMPILE_FLAG("/Wall")
+  IF(CMAKE_CXX_COMPILER_VERSION VERSION_LESS "19.0") # VS2013 and older explicitly need /arch:sse2 set, VS2015 no longer has that option, but always enabled. 
+    ADD_COMPILE_FLAG("/arch:sse2")
+  ENDIF()
+  ADD_COMPILE_FLAG("-D_CRT_SECURE_NO_WARNINGS")
```

Comment:
Could you open a bug to fix this, and remove this define? I don't think we should be using these functions :-)